src/Sql/Sql/Database/Services/AzureSqlDatabaseAdapter.cs

```diff
@@ -338,17 +340,20 @@ public void RenameDatabase(string resourceGroupName, string serverName, string d
         ///    Standard             | Standard
         ///    Basic                | Basic
         ///    Premium              | Premium
+        ///    
+        /// Also adds _S in the end of SkuName in case if it is Serverless
         /// </summary>
         /// <param name="tier">Azure Sql database edition</param>
+        /// <param name="isServerless">If sku should be serverless type</param>
         /// <returns>The sku name</returns>
-        public static string GetDatabaseSkuName(string tier)
+        public static string GetDatabaseSkuName(string tier, bool isServerless = false)
         {
             if (string.IsNullOrWhiteSpace(tier))
             {
                 return null;
             }
 
-            return SqlSkuUtils.GetVcoreSkuPrefix(tier) ?? tier;
+            return (SqlSkuUtils.GetVcoreSkuPrefix(tier) ?? tier) + (isServerless ? "_S" : "");
```

Comment:
There is already a unit test for this method, would be nice if you added `isServerless` coverage to that test.